src/routers/mailingList/types.ts

```diff
@@ -0,0 +1,5 @@
+export type MailingListRequestBody = {
+  email: string;
+  interestedEvents: string[];
+  otherFeedback: string;
+};
```

Comment:
What do you think about using the `as` keyword to try coerce each type in the route?
For example, when you extract `interestedEvents`, could you do `interestedEvents as string[]`
And if it fails at runtime (meaning the req body had something other than string[]), the user gets back a 400 because of the `try catch`?

Reply:
Casting in with `as` in TypeScript is actually all compile time so there is no check or throw at runtime.
[express-validator](https://github.com/express-validator/express-validator#documentation) might be a solution for validation. Not sure how it works with TS but I'm happy to cast once some kind of validation passes.

Reply:
Oh no way! That's interesting to know. 
Yea we need to have a good way to validate our routes.
Reading [the docs](https://express-validator.github.io/docs/) now and it looks really good.
Can you try it out for this route using like the "out of the box" set up?